tracks.yaml

```diff
@@ -1,4 +1,28 @@
 tracks:
+  kinetic:
+    actions:
+    - bloom-export-upstream :{vcs_local_uri} :{vcs_type} --tag :{release_tag} --display-uri
+      :{vcs_uri} --name :{name} --output-dir :{archive_dir_path}
+    - git-bloom-import-upstream :{archive_path} :{patches} --release-version :{version}
+      --replace
+    - git-bloom-generate -y rosrelease :{ros_distro} --source upstream -i :{release_inc}
+    - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+      -i :{release_inc} --os-name ubuntu
+    # - git-bloom-generate -y rosdebian --prefix release/:{ros_distro} :{ros_distro}
+    #   -i :{release_inc} --os-name debian --os-not-required
```

Comment:
please keep this uncommented. When we release it in kinetic we will tell the buildfarm to blacklilst the package on Debian Jessie [here](https://github.com/ros-infrastructure/ros_buildfarm_config/blob/90e681d3472b41ffc726cc2d7ede803b78ae57aa/kinetic/release-jessie-build.yaml#L14) and [here](https://github.com/ros-infrastructure/ros_buildfarm_config/blob/90e681d3472b41ffc726cc2d7ede803b78ae57aa/kinetic/release-jessie-arm64-build.yaml#L14):

Reply:
It looks like this comment hasn't been adressed

Reply:
Sorry, I am misunderstanding your commnent, "uncomment" these lines.